widgets/badgedlabel/org.eclipse.nebula.widgets.badgedlabel/src/org/eclipse/nebula/widgets/badgedlabel/BADGE_LOCATION.java

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Akuiteo (http://www.akuiteo.com). All rights reserved. This program and the
+ * accompanying materials are made available under the terms of the Eclipse
+ * Public License v1.0 which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Laurent CARON (laurent.caron at gmail dot com) - initial API and
+ * implementation
+ *******************************************************************************/
+package org.eclipse.nebula.widgets.badgedlabel;
+
+/**
+ * This enumeration describes the possible locations of the badge
+ */
+public enum BADGE_LOCATION {
+	TOP_LEFT, TOP_RIGHT, BOTTOM_LEFT, BOTTOM_RIGHT

```

Comment:
Please avoid a dedicated type here and use a combination of SWT.TOP , SWT.LEFT and so on like ControlDecoration is using.

Reply:
Done. Enumeration removed and location is based on a combination of SWT.LEFT or SWT.RIGHT/SWT.TOP or SWT.BOTTOM